internal/jobqueue/fsjobqueue/fsjobqueue.go

```diff
@@ -0,0 +1,340 @@
+// Package fsjobqueue implements a filesystem-backed job queue. It implements
+// the interfaces in package jobqueue.
+//
+// Jobs are stored in the file system, using the `jsondb` package. However,
+// this package does not use the file system as a database, but keeps some
+// state in memory. This means that access to a given directory myst be
```

Comment:
Just a small typo: 's|myst|must|'

Reply:
Unless there are other issues, let's fix this in a follow-up :)